examples/grafana/nginx-mesh-top.json

```diff
@@ -63,11 +27,9 @@
         "rgba(237, 129, 40, 0.89)",
         "#d44a3a"
       ],
-      "datasource": "${DS_PROMETHEUS}",
+      "datasource": "prometheus",
```

Comment:
Does this work with byo grafana?

Reply:
Hm, better not to change. Let me revert.

Reply:
There are more considerations here. Let me close and reopen when I've done proper testing.